packages/shared/lib/services/paginate.service.ts

```diff
@@ -110,6 +112,11 @@ class PaginationService {
                 const url: URL = new URL(nextPageLink);
                 config.endpoint = url.pathname + url.search;
             }
+
+            const apiBase = config.baseUrlOverride || template.proxy?.base_url;
+            if (config.endpoint.includes(apiBase)) {
+                config.endpoint = config.endpoint.replace(apiBase, '');
+            }
```

Comment:
not sure I follow. does that mean the returned nextPakelink contains twice the base url?

Reply:
That is correct

Reply:
😱 which provider is doing that?

Reply:
Revisited this and this was actually a result of a regex that didn't catch it correctly. Added tests to protect against this.